README.md

```diff
@@ -539,9 +539,20 @@ neutron.add_router_interface router.id, subnet.id
 
 ## Development
 
-After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+```
+$ git clone https://github.com/fog/fog-openstack.git # Clone repository
+$ cd fog-openstack; bin/setup   # Install dependencies from project directory
+$ rake spec   # Run tests
+$ bin/console   # Run interactive prompt that allows you to experiment (optional)
+$ bundle exec rake Install   # Install gem to your local machine (optional)
```

Comment:
@apophis90 should `Install` be lowercase?

Reply:
Oops. Fixed it, thx.